lib/main.coffee

```diff
@@ -11,6 +11,8 @@ Config = require './config'
 KernelManager = require './kernel-manager'
 Inspector = require './inspector'
 AutocompleteProvider = require './autocomplete-provider'
+zmq_services = require './zmq-services'
```

Comment:
The way this is done with `KernelManager` is:
- main creates an instance called `@kernelManager`
- any other classes that need the kernel manager, get a copy of `@kernelManager` through the constructor.